File: ueueo-framework/ueueo-blob-storing/src/main/java/Abp/BlobStoring/IBlobProvider.cs
using System.IO;
using System.Threading.Tasks;

namespace Volo.Abp.BlobStoring;

public interface IBlobProvider
{
    void SaveAsync(BlobProviderSaveArgs args);

    Boolean  DeleteAsync(BlobProviderDeleteArgs args);

    Boolean  ExistsAsync(BlobProviderExistsArgs args);

    StreamGetOrNullAsync(BlobProviderGetArgs args);
}

File: ueueo-frameworks/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/IBlobProvider.java
package com.ueueo.blobstoring;

import java.io.InputStream;

public interface IBlobProvider {

    void save(BlobProviderSaveArgs args);

    Boolean delete(BlobProviderDeleteArgs args);

    Boolean exists(BlobProviderExistsArgs args);

    InputStream getOrNull(BlobProviderGetArgs args);
}
